doc/source/security/index.rst

```diff
@@ -0,0 +1,85 @@
+.. _security:
+
+===============
+Security Basics
+===============
+
+By default, |PXC| does not provide any protection for stored data.
+There are several considerations to take into account for securing |PXC|:
+
+* :ref:`secure-network`
+
+  Anyone with access to your network can connect to any |PXC| node
+  either as a client or as another node joining the cluster.
+  You should consider restricting access using VPN
+  and filter traffic on ports used by |PXC|.
+
+* :ref:`encrypt-traffic`
+
+  Unencrypted traffic can potentially be viewed
+  by anyone monitoring your network.
+  You should enable encryption for all nodes in the cluster to prevent this.
+
+* Data-at-rest encryption?
+
+  Should we cover this? It seems MariaDB has it covered but nothing in PS docs.
```

Comment:
We should reframe it saying
"PXC-5.7 added support for encrypted tablespace introduced in MySQL-5.7 there-by allowing user to encrypt data at rest using MySQL keyring plugin/technique".